cernopendata/modules/fixtures/data/records/cms-derived-nanoaod-outreach-DoubleMuParked.json

```diff
@@ -0,0 +1,73 @@
+[
+  {
+    "title": "DoubleMuParked dataset from 2012 in NanoAOD format reduced on muons",
+    "publisher": "CERN Open Data Portal",
+    "recid": "12341",
+    "abstract": {
+      "description":
+            "<p>This dataset contains about 60 mio. data events from the CMS detector taken in 2012 during Run B and C. The original AOD dataset is converted to the NanoAOD format and reduced to the muon collections.</p><p>The dataset in the files is called <code>Events</code> and contains columns as described in the following table:</p><p><table style=\"width:100%\"><tr><th>Column name</th><th>Data type</th><th>Description</th></tr><tr><td><code>nMuon</code></td><td><code>unsigned int</code></td><td>Number of muons in this event</td></tr><tr><td><code>Muon_pt</code></td><td><code>float[nMuon]</code></td><td>Transverse momentum of the muons (stored as an array of size <code>nMuon</code>)</td></tr><tr><td><code>Muon_eta</code></td><td><code>float[nMuon]</code></td><td>Pseudorapidity of the muons</td></tr><tr><td><code>Muon_phi</code></td><td><code>float[nMuon]</code></td><td>Azimuth of the muons</td></tr><tr><td><code>Muon_mass</code></td><td><code>float[nMuon]</code></td><td>Mass of the muons</td></tr><tr><td><code>Muon_charge</code></td><td><code>int[nMuon]</code></td><td>Charge of the muons (either 1 or -1)</td></tr></table></p>"
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Wunsch, Stefan"
+      }
+    ],
+    "doi": "TODO: DOI_2",
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_published": "2019",
```

Comment:
How many events are in the dataset?  We can also specify NanoAOD format:

```
    "distribution": {
      "formats": [
        "root",
        "nanoaod"
      ],
      "number_events": xxxxxx,
      "number_files": 1,
      "size": 2244449133
    },
```

Reply:
Added!